tests/integration/ceph_helm_test.go

```diff
@@ -59,17 +59,19 @@ func TestCephHelmSuite(t *testing.T) {
 
 type HelmSuite struct {
 	suite.Suite
-	helper    *clients.TestClient
-	kh        *utils.K8sHelper
-	op        *TestCluster
-	namespace string
+	helper          *clients.TestClient
+	kh              *utils.K8sHelper
+	op              *TestCluster
+	namespace       string
+	rookCephCleanup bool
 }
 
 func (hs *HelmSuite) SetupSuite() {
 	hs.namespace = "helm-ns"
+	hs.rookCephCleanup = true
```

Comment:
If you're just having trouble running the helm test, you might try enabling this for another suite for local testing, then re-enable it in the helm test after you've troubleshooted locally.

Reply:
nit: How about making this a local var instead of on the HelmSuite struct? I don't see it used anywhere else.